packages/composites/token-allocation/test/unit/adapter.test.ts

```diff
@@ -105,47 +98,3 @@ describe('source adapters', () => {
     adapters.forEach((adapter) => expect(adapter.endpoints).toBeTruthy())
   })
 })
-
-describe('makeEndpointSelector', () => {
```

Comment:
Why is this test deleted?

Reply:
apparently this test is always returning true because `adapters[0].inputParameters` is undefined